src/psychrometrics/t_mrt.js

```diff
@@ -0,0 +1,162 @@
+import { valid_range } from "../utilities/utilities";
+
+const g = 9.81;
+const c_to_k = 273.15;
+const cp_air = 1004;
+
+/**
+ * Converts globe temperature reading into mean radiant temperature in accordance with either the Mixed Convection developed by Teitelbaum E. et al. (2022) or the ISO 7726:1998 Standard [5]_.
+ * @param {Number | Number[]} tg - globe temperature, [°C]
+ * @param {Number | Number[]} tdb - air temperature, [°C]
+ * @param {Number | Number[]} v - air speed, [m/s]
+ * @param {Number | Number[]} d - diameter of the globe, [m] default 0.15 m
+ * @param {Number | Number[]} emissivity - emissivity of the globe temperature sensor, default 0.95
+* @param {String} standard - {"Mixed Convection", "ISO"} either choose between the Mixed Convection and ISO formulations.
+        The Mixed Convection formulation has been proposed by Teitelbaum E. et al. (2022)
+        to better determine the free and forced convection coefficient used in the
+        calculation of the mean radiant temperature. They also showed that mean radiant
+        temperature measured with ping-pong ball-sized globe thermometers is not reliable
+        due to a stochastic convective bias [22]_. The Mixed Convection model has only
+        been validated for globe sensors with a diameter between 0.04 and 0.15 m.
+* @returns {Number | Array}
+*/
+
+export function t_mrt(
+  tg,
+  tdb,
+  v,
+  d = 0.15,
+  emissivity = 0.95,
+  standard = "Mixed Convection",
+) {
+  standard = standard.toLowerCase();
+  tdb = Array.isArray(tdb) ? tdb : [tdb];
```

Comment:
I think it is better to create a variable for `tdbIsArray` and don't convert `tdb` into an array. Then use that variable inside the map `const _tdb = tdbIsArray ? tdb[index] : tdb`
Makes it much cleared why you are doing it than checking the length of the array to be 1. Also having that `length === 1` check means that if the user gives us input that should fail like `tg = [1], tdb = [1,2]` then the function would not fail when it probably should since that input means something went wrong in the user's end, like missing data etc.